### PR TITLE
Normalize development section folders

### LINKED ISSUE #10: Create changelog
Hi, @rafaelje 

Please, create changelog file for library 

https://github.com/conventional-changelog/standard-version

### LINKED ISSUE #11: Create release
### Create releases of the library

### LINKED ISSUE #50: DevOps tasks
- [ ] Continuous integration
- [ ] Continuous deployment
- [ ] Changelog
- [ ] Test reports
- [ ] Coverage reports
- [ ] Code documentation
- [ ] Artifacts for downloads and release description

### PR DESCRIPTION
**Pull request by Naylin15**
### Changes description
Rename the folders for development section on gh-pages
Add license headers
Review doc-blocks
Modify workflow to add changelog when there is a merge into master

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [x] Docs have been added/updated

### Estimated time

|Assignee|:tomato:|
|:---|:---:|
|@Naylin15|84|

### References

Closes #10 
Closes #11 
Closes #50 
Related [#118](https://github.com/flyve-mdm/android-inventory-agent/issues/118)
Depends on #56

### CODE REVIEW

**Comment by Naylin15** [comment]
- There aren't being created unnecesary commits to develop branch, added validation for the branch
- Added new scripts to improve validation process and avoid premature and false failures, in order to keep the build status in green (readme)
- Made several tests to implement the changes

**Comment by Naylin15** [comment]
I've been having an issue with the validation on master branch, it enters a loop and creates release after release

**Comment by Naylin15** [comment]
Still making tests to solve the validation issue

File: ci/scripts/ci_changelog.sh [hunk 1]
```diff
@@ -1,60 +1,51 @@
 #!/usr/bin/env bash
 #
-#  Copyright (C) 2017 Teclib'
+#  LICENSE
 #
-#  This file is part of Flyve MDM.
+#  This file is part of Flyve MDM Agent for Android.
 #
-#  Flyve MDM Admin Dashboard Android is a subproject of Flyve MDM. Flyve MDM is a mobile
+#  Flyve MDM Agent for Android is a subproject of Flyve MDM. Flyve MDM is a mobile
```
Comment (ajsb85) [inline ci/scripts/ci_changelog.sh]: Dashboard

**Review by ajsb85 — CHANGES_REQUESTED** [review]

**Comment by Naylin15** [comment]
Fixed the validation issue

**Comment by Naylin15** [comment]
I've been having issues while sending the documentation to gh-pages

**Comment by Naylin15** [comment]
The issues are being caused due to npm

I'll make some tests using yarn

- [Error access denied](https://circleci.com/gh/Naylin15/android-mdm-dashboard-fork/989)

**Comment by Naylin15** [comment]
With yarn I avoid all previous issues

However, after running the coverage script, by some reason it turns unavailable the devdependencies, it throws a command not found

Before running the documentation, the command is found

captura de pantalla 2018-06-22 a la s 5 21 20 p m

After running the coverage script, the command is not found

captura de pantalla 2018-06-22 a la s 5 22 46 p m

**Comment by Naylin15** [comment]
I think the issue was generated due to the change of branch to update the styles for the coverage and test-reports before the execution of the other script

I'm trying to solve this by creating two new scripts

The first will generate coverage, code documentation and send them to gh-pages

The second will update headers and styles for the development folder

This is to make sure the execution of one script does not affect the other

However I'm having some issues with the permission to execute the scripts

- [Test on Circle CI](https://circleci.com/gh/Naylin15/android-mdm-dashboard-fork/1229)

captura de pantalla 2018-06-25 a la s 4 10 32 p m

**Comment by Naylin15** [comment]
There is no longer an issue with the permissions and the documentation for the development section is being created and sent as it should

**Review by ajsb85 — DISMISSED** [review]

**Review by ajsb85 — DISMISSED** [review]

**Comment by Naylin15** [comment]
Hi, @ajsb85 @rafaelje 

The PR is finished, this is how the new workflow will be:

### On feature branch

- run Build
- run Test Api v16

### On develop

There are two main scripts which include the commit validation to avoid repeated workflows, and contain the scripts related to their process.

- Setup environment (ci_setup.sh)
- Deploy Beta (ci_deploy_beta.sh)
  - Update version and code (ci_updateversion.sh)
  - Create about information (ci_about.sh)
  - Transifex for translations (ci_transifex.sh)
  - Create APK
  - Deploy to Google Play Beta version (ci_fastlane.sh)
- Documentation (ci_documentation.sh)
  - Create Coverage, Test reports and Code Documentation (ci_generate_documentation.sh)
  - Update headers and styles for proper display on project site
- Update screenshots (ci_screenshots.sh)

### On master

There is one main script that includes the commit validation to avoid repeated workflows, and contains the scripts related to the process.

- Setup environment (ci_setup.sh)
- Deploy Production (ci_deploy_production.sh)
  - Update version and code, generate Changelog (ci_updateversion.sh)
  - Create about information (ci_about.sh)
  - Transifex for translations (ci_transifex.sh)
  - Create APK
  - Push new tags and update develop branch (ci_push_changes.sh)
  - Deploy to Google Play (ci_fastlane.sh)
  - Create a GitHub release (ci_github_release.sh)


## Note:

I moved the validation because in the old workflow it gave false errors, when actually the build was successful

Some scripts were removed since the use of the gh-pages library simplified the process and were no longer necessary

On develop branch there will be no commits when making merge with it, but it will be updated the android code and about properties, and only the documentation and the screenshots will be generated - updated

On master branch there will be commits regarding the changelog, the update on Android Manifest values and about

Also, I'll [set up this project on Circle CI](https://circleci.com/add-projects/gh/flyve-mdm) with the variables from the android-mdm-agent and the GITHUB_EMAIL from #56 

captura de pantalla 2018-06-27 a la s 9 34 51 a m

**Comment by rafaelje** [comment]
@ajsb85 @Naylin15 made important changes and organize the CI scripts IMO please add your name on @author tag in the license header.

**Review by ajsb85 — DISMISSED** [review]

**Review by ajsb85 — APPROVED** [review]

**Pull request merged**